src/main/webapp/app/pages/curation/button/AddMutationButton.tsx

```diff
@@ -7,9 +7,11 @@ const AddMutationButton: React.FunctionComponent<{
   onClickHandler: (show: boolean) => void;
   showIcon?: boolean;
   showFullTitle?: boolean;
-}> = ({ showAddMutationModal, onClickHandler, showIcon = true, showFullTitle = false }) => {
+  disabled?: boolean;
```

Comment:
I would keep it consistent with the `RCTButton` and just inject the CurationPage store into `AddMutationButton` and use the `readOnly` property for disabled. Same end result, but one less prop to pass in.